### ISSUE TITLE
FR: support for alldayDueDate

### ISSUE THREAD

**Issue by chrisgrieser** [keith/reminders-cli#79]
The reminders.app supports allday due dates, ie.e. due dates without a specific date. Using JXA, we can create such like this:

```js
const today = new Date();
const newReminder = Application("Reminders").Reminder({ 
    name: input.trim(), 
    alldayDueDate: today
});
Application("Reminders").defaultList().reminders.push(newReminder);
```

When using the reminders-cli, and add a new reminder:
```bash
reminders add MyList--due-date=today "foobar"
```
It always also sets `00:00` as due time. Explicitly using a date without a time, e.g. `--due-date=2024-03-08` also results in a due time of `00:00`.

**Comment by keith** [comment]
included in 2.5.1

**Comment by chrisgrieser** [comment]
Thank you
